lib/wsdl.js

```diff
@@ -116,6 +117,12 @@ var Element = function(nsName, attrs, options) {
       }
     }
   }
+  for (var schemaKey in schemaAttrs) {
+    var schemaMatch = /^xmlns:?(.*)$/.exec(schemaKey);
+    if (schemaMatch && schemaMatch[1] && schemaMatch) {
```

Comment:
did you really want to check for `schemaMatch` twice here? 😄

Reply:
fixed :)